app/e2e/client/tabbar.ts

```diff
@@ -5,10 +5,11 @@ import { addAction } from '../../../client/views/room/lib/Toolbox';
 import { useSetting } from '../../../client/contexts/SettingsContext';
 import { usePermission } from '../../../client/contexts/AuthorizationContext';
 import { useMethod } from '../../../client/contexts/ServerContext';
+import { e2e } from './rocketchat.e2e';
 
 addAction('e2e', ({ room }) => {
 	const e2eEnabled = useSetting('E2E_Enable');
-	const hasPermission = usePermission('edit-room', room._id);
+	const hasPermission = usePermission('edit-room', room._id) && usePermission('toggle-room-e2e-encryption', room._id) && (e2e.isReady() || room.encrypted);
```

Comment:
I just found out this is actually breaking for DMs.. since there is no owner on a DM, no one has this permission

Reply:
A wise man once said: "If nobody is the owner, everybody is an owner"

should I let anybody toggle it on DMs? (even multi-party DMs)

Reply:
yes. everybody in a DM (even mutl-party) should be able to toggle it. not sure sure if we should make them all owners or have an special condition